packages/core/admin/ee/admin/src/content-manager/pages/EditView/components/tests/AssigneeSelect.test.tsx

```diff
@@ -25,16 +26,17 @@ describe('EE | Content Manager | EditView | InformationBox | AssigneeSelect', ()
   });
 
   it('renders a select with users, none is selected', async () => {
-    const { getByRole, getByText, queryByText, user } = render(<AssigneeSelect />);
+    const { getByRole, queryByText, user, findByText } = render(<AssigneeSelect />);
 
     await waitFor(() => expect(queryByText('John Doe')).not.toBeInTheDocument());
 
     await user.click(getByRole('combobox'));
 
-    await waitFor(() => expect(getByText('John Doe')).toBeInTheDocument());
+    await findByText('John Doe');
```

Comment:
Seems like you are not doing any assertion with that element, is it wanted?

Reply:
Same question for the other tests 🤔

Reply:
Find by text fails if it's not found, the eslint plugin suggests making these changes which I guess it makes sense since it's a sort of assertion...

Reply:
That's what I thought, in terms of error messages, is it understandable if the test fails at this level?

Reply:
Afaik yes, because it will complain it can't find the element, similar to an assertion.